### ISSUE TITLE
Do I need additional steps to use ArrayField?

### ISSUE THREAD

**Issue by msdos** [coleifer/peewee#1817]
I'm trying to use the `ArrayField` from Postgresql but I can't use your `get` method. Here is a proof of concept of the problem I'm facing it (I adapted the sample code from an old issue here in the repository):

```python
#!/usr/bin/env python

from peewee import Model
from peewee import TextField
from playhouse.postgres_ext import ArrayField
from playhouse.postgres_ext import PostgresqlExtDatabase
from playhouse.shortcuts import model_to_dict


db = PostgresqlExtDatabase(
    'peewee_test',
    user='postgres',
    password='postgres',
    host='localhost',
    autorollback=True
)


class BaseModel(Model):
    class Meta:
        database = db


class BlogPost(BaseModel):
    title = TextField()
    tags = ArrayField(TextField)


def main():
    db.create_tables([BlogPost])
    BlogPost.delete().execute()
    BlogPost.create(
        title='awesome',
        tags=['foo', 'bar', 'baz'])
    BlogPost.create(
        title='bad',
        tags=['foo', 'bax', 'baz'])
    for b in BlogPost.select().where(BlogPost.tags.contains_any('python', 'bar')):  # NOQA
        print(b.title)

    """SQL so far:
    SELECT tablename FROM pg_catalog.pg_tables WHERE schemaname = %s ORDER BY tablename
    DELETE FROM "blogpost"
    INSERT INTO "blogpost" ("title", "tags") VALUES (%s, %s) RETURNING "blogpost"."id"
    INSERT INTO "blogpost" ("title", "tags") VALUES (%s, %s) RETURNING "blogpost"."id"
    SELECT "t1"."id", "t1"."title", "t1"."tags" FROM "blogpost" AS "t1" WHERE ("t1"."tags" && %s::TEXT[])
    """

    model_dict = model_to_dict(b)
    print(model_dict)
    b = BlogPost.get(**model_dict)

    """SQL so far:
    SELECT "t1"."id", "t1"."title", "t1"."tags" FROM "blogpost" AS "t1" WHERE ((("t1"."id" = %s) AND ("t1"."tags" = (%s, %s, %s))) AND ("t1"."title" = %s)) LIMIT %s OFFSET %s

peewee.py", line 2712, in execute_sql
    cursor.execute(sql, params or ())
psycopg2.ProgrammingError: operator does not exist: text[] = record
LINE 1: ... AS "t1" WHERE ((("t1"."id" = 3) AND ("t1"."tags" = (ARRAY['...
                                                             ^
HINT:  No operator matches the given name and argument type(s). You might need to add explicit type casts.    
"""


if __name__ == '__main__':
    main()
```

I'm using Postgresql 10.6, Python 3.6.7 and peewee 3.8.0 (psycopg2 is 2.7.6.1)

**Comment by coleifer** [comment]
See 04aa7a0 for the fix.

**Comment by msdos** [comment]
Is this planned to a 3.8.1 release? The fix was really fast, thanks.

**Comment by coleifer** [comment]
I'll release this fix along with 3.8.1 at some point in the coming days/weeks.

In the meantime there are a number of workarounds. The simplest being instead of using `.get()` with keyword arguments, use expressions, e.g.: `MyModel.foo == 'bar'` is *always* preferable to `foo='bar'`.
